cmu_graphics/__init__.py

```diff
@@ -1,7 +1,13 @@
 import __main__
 import os
 
-main_directory = os.path.dirname(os.path.abspath(__main__.__file__))
+try:
+    main_directory = os.path.dirname(__main__.__file__)
+except Exception:
+    main_directory = os.path.dirname(os.path.abspath(__main__.__name__))
+    print(f"""Attention:
```

Comment:
Though I can see how this could be helpful in some situations, I think this warning will mostly be noise to users, and we should remove it.